src/main/java/org/terasology/las/platform/FloatingPlatformProvider.java

```diff
@@ -38,28 +36,32 @@
-    private static final Rect2i FLOATING_PLATFORM_REGION = Rect2i.createFromMinAndMax(LASUtils.FLOATING_PLATFORM_POSITION.getX() - LASUtils.FLOATING_PLATFORM_WIDTH / 2,
-            LASUtils.FLOATING_PLATFORM_POSITION.getZ() - LASUtils.FLOATING_PLATFORM_LENGTH / 2,
-            LASUtils.FLOATING_PLATFORM_POSITION.getX() + LASUtils.FLOATING_PLATFORM_WIDTH / 2,
-            LASUtils.FLOATING_PLATFORM_POSITION.getZ() + LASUtils.FLOATING_PLATFORM_LENGTH /2);
+    private static final Rect2i FLOATING_PLATFORM_REGION =
+            Rect2i.createFromMinAndMax(LASUtils.FLOATING_PLATFORM_POSITION.x() - LASUtils.FLOATING_PLATFORM_WIDTH / 2,
+                    LASUtils.FLOATING_PLATFORM_POSITION.z() - LASUtils.FLOATING_PLATFORM_LENGTH / 2,
+                    LASUtils.FLOATING_PLATFORM_POSITION.x() + LASUtils.FLOATING_PLATFORM_WIDTH / 2,
+                    LASUtils.FLOATING_PLATFORM_POSITION.z() + LASUtils.FLOATING_PLATFORM_LENGTH / 2);
 
-    private static final Region3i RED_TELEPORTER_REGION = Region3i.createFromMinMax(new Vector3i(LASUtils.FLOATING_PLATFORM_POSITION.getX() + LASUtils.TELEPORTER_OFFSET,
-                                                                                            LASUtils.FLOATING_PLATFORM_POSITION.getY() + 1,
-                                                                                            LASUtils.FLOATING_PLATFORM_POSITION.getZ() + LASUtils.NPC_OFFSET),
-                                                                                    new Vector3i(LASUtils.FLOATING_PLATFORM_POSITION.getX() + LASUtils.TELEPORTER_OFFSET,
-                                                                                            LASUtils.FLOATING_PLATFORM_POSITION.getY() + 1,
-                                                                                            LASUtils.FLOATING_PLATFORM_POSITION.getZ() + LASUtils.NPC_OFFSET));
-    private static final Region3i BLACK_TELEPORTER_REGION = Region3i.createFromMinMax(new Vector3i(LASUtils.FLOATING_PLATFORM_POSITION.getX() - LASUtils.TELEPORTER_OFFSET,
-                                                                                            LASUtils.FLOATING_PLATFORM_POSITION.getY() + 1,
-                                                                                            LASUtils.FLOATING_PLATFORM_POSITION.getZ() + LASUtils.NPC_OFFSET),
-                                                                                        new Vector3i(LASUtils.FLOATING_PLATFORM_POSITION.getX() - LASUtils.TELEPORTER_OFFSET,
-                                                                                                LASUtils.FLOATING_PLATFORM_POSITION.getY() + 1,
-                                                                                                LASUtils.FLOATING_PLATFORM_POSITION.getZ() + LASUtils.NPC_OFFSET));
+    private static final BlockRegion RED_TELEPORTER_REGION =
```

Comment:
as this is a constant it should be exposed as 
```suggestion
    private static final BlockRegionc RED_TELEPORTER_REGION =
```